python/src/uagents/mailbox.py

```diff
@@ -24,6 +24,7 @@
 class AgentverseConnectRequest(Model):
     user_token: str
     agent_type: AgentType
+    endpoint: Optional[str] = None
```

Comment:
Changed this to a string to simplify things, because it gets to complicated to next a pydantic v2 model inside of a pydantic v1 model.

Reply:
Can it only be a single endpoint or several?

Reply:
Just a single endpoint now.